Gopkg.toml

```diff
@@ -17,7 +17,13 @@ required = [
   "knative.dev/test-infra/scripts",
   "knative.dev/test-infra/tools/dep-collector",
   # For load testing.
-  "github.com/tsenart/vegeta"
+  "github.com/tsenart/vegeta",
+  # For istio client-go apis.
```

Comment:
Why do we need to add them to this block? Doesn't our code depend on them anyway?

Reply:
Right, those are not really needed. Thanks!